docs/price-feed/index.adoc

```diff
@@ -40,12 +40,13 @@ when 1 ether becomes worth 10,000,000,000x as much as 1 bitcoin, we find unlikel
 Rather than prematurely optimize, incorporating a 2 integer ratio of `x` wei to `y` satoshi and changing the call semantics,
 we leave this as a future exercise for governance.
 
-NOTE: The Medianizer in user for tBTC v1 is what Maker calls an ETHBTC feed,
+NOTE: The Medianizer in used for tBTC v1 is what Maker calls an ETHBTC feed,
```

Comment:
Heh "in use" or "used". I'm sure my last comment was wrong 😅

Reply:
Oh ffs, no, this was me somehow completely missing what I was doing.